forest/benchmarking/direct_fidelity_estimation.py

```diff
@@ -172,7 +172,7 @@ def _monte_carlo_dfe(program: Program, qubits: Sequence[int], in_states: list, n
         )
 
 def monte_carlo_state_dfe(program: Program, qubits: List[int], benchmarker: BenchmarkConnection,
-                            n_terms: int = 200) -> TomographyExperiment:
+                          n_terms=200) -> TomographyExperiment:
```

Comment:
This was probably unintentional, but I'm ok to merge as is. We will need to do a pass through the whole repo for these things anyways.